src/buf.ts

```diff
@@ -1,5 +1,26 @@
 import * as child_process from "child_process";
 import { Error } from "./errors";
+import { parse, Version } from "./version";
+
+// Don't defined latest version right now, adds a lot
+// of overhead to maintain.
+/*
```

Comment:
nit: generally do not prefer c-style comments, VSCode should have some plugin where you can comment out with `//` for a selection of code I think

Reply:
Ha, I wasn't sure if I was gonna get pulled up on this, but duly noted 😁.